ui/design-system-core/src/components/AppAvatar/index.tsx

```diff
@@ -51,15 +51,15 @@ const AppAvatar: React.FC<IAppAvatarProps> = props => {
     }.webp`;
   }
 
-  const className = `box-border cursor-${
+  const className = apply`box-border cursor-${
     isClickable ? 'pointer' : 'default'
   } select-none relative overflow-hidden w-${avatarSizesMap[size]} h-${
     avatarSizesMap[size]
   } rounded-full bg-${backgroundColor ? backgroundColor : 'white'} border-${
     border ? avatarBorderSizesMap[border] : '0'
   } border-${borderColor ? avatarBorderColorsMap[borderColor] : 'transparent'}`;
 
-  const activeOverlayClass = 'bg-grey6 opacity-25 z-10 absolute top-0 left-0 w-full h-full';
+  const activeOverlayClass = apply('bg-grey6 opacity-25 z-10 absolute top-0 left-0 w-full h-full');
```

Comment:
Isn't it beneficial to group all styles into one class so that we can support style [overrides](https://twind.dev/handbook/styling-with-twind.html#the-apply-function) ? I only see a benefit by using apply. 

And as for cx let's say we have a use case where we want to apply white background when a certain flag is on and we can express it like this :

`cx({ 'bg-white': isTrue, rounded })`

In our case we are used to doing ternary operator such that the above can be expressed as:
`rounded ${isTrue ? 'bg-white': ''}`

That scenario imo is the right use case for cx. Other than that for two or more classes we need to the `apply` function.

Reply:
In addition, I see `cx` has a use case for grouping : https://twind.style/grouping-syntax